apps/native/app/src/screens/health/health-overview.tsx

```diff
@@ -187,7 +206,11 @@ export const HealthOverviewScreen: NavigationFunctionComponent = ({
               isUtilityButton
               icon={externalLinkIcon}
               iconStyle={{ tintColor: theme.color.dark300 }}
-              style={{ flex: 1 }}
+              style={{
+                flex: 1,
+                maxWidth: width * 0.5 - theme.spacing[3],
+                minWidth: width * 0.5 - theme.spacing[3],
```

Comment:
Both these styles are needed to handle both cases if Vaccinations button is shown or not. (If it is shown then this is the third button and is in a separate row so we want to make sure it is not bigger than 50% of the space - if the vaccinations button is not shown we want to make sure it is not smaller than 50% of the space)